### ISSUE TITLE
Dependencies not usable in 16.0.2

### ISSUE THREAD

**Issue by soniar4i** [googleads/google-ads-java#543]
**Describe the bug:**
When changing from google-ads:16.0.1 to google-ads:16.0.2, libraries like protobuf-java or gax are not available to be used in our applications. It seems that [16.0.2](https://github.com/googleads/google-ads-java/releases/tag/16.0.2) doesn't really expose these libraries making imposible to use things like gax.rpc.ServerStream or protobuf.FieldMask in our projects.

**Steps to Reproduce:**
Add `implementation 'com.google.api-ads:google-ads:16.0.2'` to any project and try to import any class like `import com.google.protobuf.FieldMask;` you won't be able to use it.

**Expected behavior:**
Being able to import without any problem.

**Client library version and API version:**
Client library version: 16.0.2
Google Ads API version: V9
JDK version: 16

**Comment by manudevelopia** [comment]
Same here, gradle build fails 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.google.api:gax:.
     Required by:
         project : > com.google.api-ads:google-ads:16.0.2
   > Could not find com.google.api:gax-grpc:.
     Required by:
         project : > com.google.api-ads:google-ads:16.0.2
   > Could not find io.grpc:grpc-stub:.
     Required by:
         project : > com.google.api-ads:google-ads:16.0.2
   > Could not find io.grpc:grpc-protobuf:.
     Required by:
         project : > com.google.api-ads:google-ads:16.0.2
   > Could not find com.google.auth:google-auth-library-oauth2-http:.
     Required by:
         project : > com.google.api-ads:google-ads:16.0.2
```

**Comment by nwbirnie** [comment]
This is already fixed in main, will be in the next release. In the meantime as a workaround please can you add the following dependency:

`implementation platform('com.google.cloud:google-cloud-shared-dependencies:2.4.0')`

**Comment by soniar4i** [comment]
Cool! Thanks a lot.

**Comment by jradcliff** [comment]
FYI, we just released 16.0.3 with the fixes @nwbirnie mentioned.

https://github.com/googleads/google-ads-java/releases/tag/16.0.3
